src/main/kotlin/com/example/noswimmingserver/domain/teacher/presentation/TeacherController.kt

```diff
@@ -26,4 +30,9 @@ class TeacherController(
     fun editTeacherInfo(@RequestBody @Valid request: UpdateTeacherInfoRequest) {
         updateTeacherInfoService.execute(request)
     }
+
+    @PostMapping("/token")
+    fun teacherSignIn(@RequestBody @Valid request: TeacherSignInRequest): TokenResponse {
```

Comment:
🚫 **[ktlint]** <annotation> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Multiple annotations should not be placed on the same line as the annotated construct